package.json

```diff
@@ -18,30 +22,31 @@
     "homepage": "https://github.com/GPII/gpii-express-user",
     "dependencies": {
         "foundation-sites": "6.3.1",
-        "gpii-binder": "1.0.3",
-        "gpii-express": "1.0.14",
-        "gpii-handlebars": "1.1.0",
-        "gpii-json-schema": "1.0.5",
-        "infusion": "3.0.0-dev.20180326T173646Z.8c6a109b1",
-        "kettle": "1.7.3",
-        "markdown-it": "8.4.1",
-        "nodemailer": "4.6.7",
+        "gpii-binder": "1.0.5",
+        "gpii-express": "1.0.15",
+        "gpii-handlebars": "2.1.0-dev.20191014T141924Z.45a74ef.GPII-4100",
+        "gpii-json-schema": "2.1.2-dev.20191014T143306Z.8a372f7.GPII-4022",
+        "infusion": "3.0.0-dev.20191009T141140Z.32c9263b4.FLUID-6148",
+        "kettle": "1.11.0",
+        "markdown-it": "10.0.0",
+        "nodemailer": "6.3.0",
         "nodemailer-smtp-transport": "2.7.4",
-        "request": "2.87.0"
+        "request": "2.88.0"
     },
     "devDependencies": {
-        "cookie": "0.3.1",
-        "eslint": "5.1.0",
+        "cookie": "0.4.0",
         "eslint-config-fluid": "1.3.0",
-        "gpii-grunt-lint-all": "1.0.1-dev.20180706T153657Z.4cbbd61",
-        "gpii-mail-test": "1.0.3",
-        "gpii-pouchdb": "1.0.12",
-        "gpii-webdriver": "1.0.2",
-        "grunt": "1.0.3",
-        "handlebars": "4.0.11",
-        "istanbul": "0.4.5",
-        "mailparser": "0.6.2",
+        "eslint": "6.5.0",
+        "gpii-couchdb-test-harness": "1.0.0-dev.20190627T080736Z.0e3a1fd.GPII-3531",
+        "gpii-grunt-lint-all": "1.0.5",
+        "gpii-mail-test": "1.0.5",
+        "gpii-webdriver": "1.1.0",
+        "grunt": "1.0.4",
```

Comment:
A few of these have become further outdated since this pull was issued, e.g. grunt -> 1.1.0, eslint -> 6.8.0 (doesn't lint-all pull in its own deps anyway?)
Also please could you try updating to kettle 1.12.0 and infusion 3.0.0-dev.20200317T122331Z.1aa7ea8a1.FLUID-6145 - thanks